docs/content/Select.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Select
+description: Use a select input when a user needs to select one option from a long list
+status: Alpha
+source: https://github.com/primer/react/blob/main/src/Select.tsx
+storybook: '/react/storybook?path=/story/forms-select--default'
+---
+
+import {ComponentChecklist} from '../src/component-checklist'
+import {Select, Text} from '@primer/react'
+
+## Examples
+
+### Basic
+
+```jsx live
+<>
+  <Text fontWeight="bold" fontSize={1} as="label" display="block" htmlFor="basic">
+    Preferred Primer component interface
+  </Text>
+  <Select id="basic">
+    <Select.Option value="figma">Figma</Select.Option>
+    <Select.Option value="css">Primer CSS</Select.Option>
+    <Select.Option value="prc">Primer React components</Select.Option>
+    <Select.Option value="pvc">Primer ViewComponents</Select.Option>
+  </Select>
+</>
+```
+
+### With grouped options
+
+```jsx live
+<>
+  <Text fontWeight="bold" fontSize={1} as="label" display="block" htmlFor="grouped">
+    Preferred Primer component interface
+  </Text>
+  <Select id="grouped">
+    <Select.OptGroup label="GUI">
+      <Select.Option value="figma">Figma</Select.Option>
+    </Select.OptGroup>
+    <Select.OptGroup label="Code">
+      <Select.Option value="css">Primer CSS</Select.Option>
+      <Select.Option value="prc">Primer React components</Select.Option>
+      <Select.Option value="pvc">Primer ViewComponents</Select.Option>
+    </Select.OptGroup>
+  </Select>
+</>
+```
+
+### With a placeholder
+
+```jsx live
+<>
+  <Text fontWeight="bold" fontSize={1} as="label" display="block" htmlFor="withPlaceholder">
+    Preferred Primer component interface
+  </Text>
+  <Select id="withPlaceholder" placeholder="Pick an interface">
+    <Select.Option value="figma">Figma</Select.Option>
+    <Select.Option value="css">Primer CSS</Select.Option>
+    <Select.Option value="prc">Primer React components</Select.Option>
+    <Select.Option value="pvc">Primer ViewComponents</Select.Option>
+  </Select>
+</>
+```
+
+## Props
+
+### Select
+
+The `Select` component accepts the same props as a native HTML [`<select>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select) except for `multiple` and `size`.
+
+### Select.OptGroup
+
+The `Select.OptGroup` component accepts the same props as a native HTML [`<optgroup>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/optgroup).
+
+### Select.Option
+
+The `Select.Option` component accepts the same props as a native HTML [`<option>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option).
+
+<Props of={Select.Option} />
```

Comment:
This doesn't seem to be working
![Screenshot 2021-12-17 at 14 36 54](https://user-images.githubusercontent.com/13340707/146560757-b0e44113-e625-432e-a9bc-36da2b11cda8.png)